en/docs/includes/setting-up-workspace-for-mi-for-vscode.md

```diff
@@ -5,4 +5,4 @@ The following software and configurations are required to proceed with this tuto
 - **Apache Maven:** Ensure Apache Maven is installed and its path is correctly set within the system's PATH environment variable.
 
 !!! info
-    Follow the [Install Micro Integrator for VS Code](https://mi.docs.wso2.com/en/4.3.0/develop/mi-for-vscode/install-wso2-mi-for-vscode) documentation for a complete installation guide.
+    Follow the [Install Micro Integrator for VS Code](https://mi.docs.wso2.com/en/4.4.0/develop/mi-for-vscode/install-wso2-mi-for-vscode) documentation for a complete installation guide.
```

Comment:
Can't we use `{{base_path}}` instead of `https://mi.docs.wso2.com/en/4.4.0/` ? Please check other places as well.

Reply:
It doesn't work for the includes section.